07-memory/kernel_space/result.txt

```diff
@@ -0,0 +1,265 @@
+[29608.336266] 37           137438953472        274        290        386        243        259        352
+[29608.336369] 38           274877906944        273        289        379        243        253        330
+[29608.336472] 39           549755813888        276        288        368        243        252        324
+[29608.336658] 40          1099511627776        272        354       1035        243        286        627
+[29608.336769] 41          2199023255552        274        293        434        240        261        404
+[29608.336874] 42          4398046511104        270        294        407        244        253        324
+[29608.336977] 43          8796093022208        274        289        406        243        252        313
+[29608.337080] 44         17592186044416        272        292        419        240        251        316
+[29608.337184] 45         35184372088832        276        292        407        241        255        319
+[29608.337289] 46         70368744177664        275        297        454        243        257        357
+[29608.337392] 47        140737488355328        273        294        420        243        255        344
+[29608.337495] 48        281474976710656        275        297        464        243        253        334
+[29608.337653] 49        562949953421312        274        349        963        242        280        578
+[29608.337782] 50       1125899906842624        277        302        496        243        262        411
+[29608.337887] 51       2251799813685248        273        289        379        241        255        365
+[29608.337990] 52       4503599627370496        274        293        433        244        253        315
+[29608.338094] 53       9007199254740992        275        296        450        243        253        333
+[29608.338197] 54      18014398509481984        273        293        414        243        252        309
+[29608.338300] 55      36028797018963968        276        290        390        243        254        312
+[29608.338403] 56      72057594037927936        275        286        357        244        252        314
+[29608.338505] 57     144115188075855872        273        286        368        244        255        331
+[29608.338665] 58     288230376151711744        273        351        975        242        284        630
+[29608.338795] 59     576460752303423488        275        307        554        245        266        427
+[29608.338900] 60    1152921504606846976        273        287        369        239        255        344
+[29608.339004] 61    2305843009213693952        273        295        431        243        254        325
+[29608.339108] 62    4611686018427387904        278        294        413        244        256        348
+[29608.339212] 63    9223372036854775808        277        296        438        246        256        325
```

Comment:
This lines looks strange. Is it possible to allocate bigger buffer. What is the upper limit of allocated memory in your system. I suppose that the program has some error in memory allocation.